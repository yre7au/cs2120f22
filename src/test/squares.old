---- MODULE squares  ----
EXTENDS TLC, Integers

(*--algorithm squares
variables
    x \in 1..11;

begin
    assert x ^ 2 <= 100;
end algorithm; *)
====
