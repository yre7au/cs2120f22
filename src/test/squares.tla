---- MODULE squares  ----
EXTENDS TLC, Integers

(*--algorithm squares
variables
    x \in 1..11;

begin
    assert x ^ 2 <= 100;
end algorithm; *)
\* BEGIN TRANSLATION (chksum(pcal) = "ba67d89f" /\ chksum(tla) = "cea41459")
VARIABLES x, pc

vars == << x, pc >>

Init == (* Global variables *)
        /\ x \in 1..11
        /\ pc = "Lbl_1"

Lbl_1 == /\ pc = "Lbl_1"
         /\ Assert(x ^ 2 <= 100, "Failure of assertion at line 9, column 5.")
         /\ pc' = "Done"
         /\ x' = x

(* Allow infinite stuttering to prevent deadlock on termination. *)
Terminating == pc = "Done" /\ UNCHANGED vars

Next == Lbl_1
           \/ Terminating

Spec == Init /\ [][Next]_vars

Termination == <>(pc = "Done")

\* END TRANSLATION 
====
