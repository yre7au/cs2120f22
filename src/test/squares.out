@!@!@STARTMSG 2262:0 @!@!@
TLC2 Version 2.16 of 31 December 2020 (rev: cdddf55)
@!@!@ENDMSG 2262 @!@!@
@!@!@STARTMSG 2187:0 @!@!@
Running breadth-first search Model-Checking with fp 120 and seed 3426390570086903495 with 1 worker on 16 cores with 7120MB heap and 64MB offheap memory [pid: 1907] (Linux 5.10.102.1-microsoft-standard-WSL2 amd64, Ubuntu 11.0.16 x86_64, MSBDiskFPSet, DiskStateQueue).
@!@!@ENDMSG 2187 @!@!@
@!@!@STARTMSG 2220:0 @!@!@
Starting SANY...
@!@!@ENDMSG 2220 @!@!@
Parsing file /workspaces/cs2120f22/src/test/squares.tla
Labels added.
Parsing file /tmp/TLC.tla
Parsing file /tmp/Integers.tla
Parsing file /tmp/Naturals.tla
Parsing file /tmp/Sequences.tla
Parsing file /tmp/FiniteSets.tla
Semantic processing of module Naturals
Semantic processing of module Sequences
Semantic processing of module FiniteSets
Semantic processing of module TLC
Semantic processing of module Integers
Semantic processing of module squares
@!@!@STARTMSG 2219:0 @!@!@
SANY finished.
@!@!@ENDMSG 2219 @!@!@
@!@!@STARTMSG 2185:0 @!@!@
Starting... (2022-10-05 03:08:45)
@!@!@ENDMSG 2185 @!@!@
@!@!@STARTMSG 2189:0 @!@!@
Computing initial states...
@!@!@ENDMSG 2189 @!@!@
@!@!@STARTMSG 2269:0 @!@!@
Computed 2 initial states...
@!@!@ENDMSG 2269 @!@!@
@!@!@STARTMSG 2269:0 @!@!@
Computed 4 initial states...
@!@!@ENDMSG 2269 @!@!@
@!@!@STARTMSG 2269:0 @!@!@
Computed 8 initial states...
@!@!@ENDMSG 2269 @!@!@
@!@!@STARTMSG 2190:0 @!@!@
Finished computing initial states: 11 distinct states generated at 2022-10-05 03:08:45.
@!@!@ENDMSG 2190 @!@!@
@!@!@STARTMSG 2132:1 @!@!@
The first argument of Assert evaluated to FALSE; the second argument was:
"Failure of assertion at line 9, column 5."
@!@!@ENDMSG 2132 @!@!@
@!@!@STARTMSG 2121:1 @!@!@
The behavior up to this point is:
@!@!@ENDMSG 2121 @!@!@
@!@!@STARTMSG 2217:4 @!@!@
1: <Initial predicate>
/\ x = 11
/\ pc = "Lbl_1"

@!@!@ENDMSG 2217 @!@!@
@!@!@STARTMSG 2103:1 @!@!@
The error occurred when TLC was evaluating the nested
expressions at the following positions:
0. Line 20, column 10 to line 23, column 18 in squares
1. Line 20, column 13 to line 20, column 24 in squares
2. Line 21, column 13 to line 21, column 77 in squares


@!@!@ENDMSG 2103 @!@!@
@!@!@STARTMSG 2201:0 @!@!@
The coverage statistics at 2022-10-05 03:08:45
@!@!@ENDMSG 2201 @!@!@
@!@!@STARTMSG 2773:0 @!@!@
<Init line 16, col 1 to line 16, col 4 of module squares>: 11:11
@!@!@ENDMSG 2773 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 17, col 12 to line 17, col 22 of module squares: 1
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 18, col 12 to line 18, col 23 of module squares: 11
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<Lbl_1 line 20, col 1 to line 20, col 5 of module squares>: 10:10
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 20, col 13 to line 20, col 24 of module squares: 22
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 20, col 13 to line 20, col 14 of module squares: 12
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 21, col 13 to line 21, col 77 of module squares: 10
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  |line 21, col 20 to line 21, col 31 of module squares: 12
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 22, col 13 to line 22, col 24 of module squares: 10
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 23, col 13 to line 23, col 18 of module squares: 10
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2772:0 @!@!@
<Terminating line 26, col 1 to line 26, col 11 of module squares>: 0:0
@!@!@ENDMSG 2772 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 26, col 16 to line 26, col 26 of module squares: 10
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2221:0 @!@!@
  line 26, col 31 to line 26, col 44 of module squares: 0
@!@!@ENDMSG 2221 @!@!@
@!@!@STARTMSG 2202:0 @!@!@
End of statistics.
@!@!@ENDMSG 2202 @!@!@
@!@!@STARTMSG 2200:0 @!@!@
Progress(2) at 2022-10-05 03:08:45: 21 states generated (2,857 s/min), 21 distinct states found (2,857 ds/min), 10 states left on queue.
@!@!@ENDMSG 2200 @!@!@
@!@!@STARTMSG 2199:0 @!@!@
21 states generated, 21 distinct states found, 10 states left on queue.
@!@!@ENDMSG 2199 @!@!@
@!@!@STARTMSG 2186:0 @!@!@
Finished in 448ms at (2022-10-05 03:08:45)
@!@!@ENDMSG 2186 @!@!@
